spec.md

```diff
@@ -1125,6 +1141,11 @@ message ControllerServiceCapability {
       // with the snapshot_id as the filter to query whether the
       // uploading process is complete or not.
       LIST_SNAPSHOTS = 6;
+      // Plugins supporting efficient volume cloning MAY report this
+      // capability.  Source volume must be managed by the same plugin.
+      // Not all volume sources and parameters combinations MAY work.
+      // CLONE_VOLUME is NOT REQUIRED.
```

Comment:
"... is NOT REQUIRED" seems redundant. It's a capability field *because* it's optional. I don't think there's a need to state that here.

Reply:
I agree, I just copied from the CREATE_DELETE_SNAPSHOT in case this was something that should be done in the specs.  Removing it.